### PR TITLE
Enable integration test framework and running of jetty-console

### PR DESCRIPTION
**Pull request by awoods**
This update depends on the merge of: https://github.com/fcrepo4-labs/jetty-console/pull/1

An improvement to this PR would be to refactor the pom.xml dependency updates in the top-level "dependencyManagement" section.

**JIRA Ticket**:
https://jira.lyrasis.org/browse/FCREPO-3153

# What does this Pull Request do?
- Updates dependencies
- Allows jetty-console to run

# How should this be tested?
- Build the project with the "one-click" profile
```
mvn clean install -Pone-click
```
- Run the one-click app
```
java -jar fcrepo-webapp/target/fcrepo-webapp-6.0.0-SNAPSHOT-jetty-console.jar --headless
```
   - Observe a running Fedora at port 8080
   - Note: Since many of the Injected components are not yet available, the application fails on any UI action

# Interested parties
Tag @fcrepo4/committers

### CODE REVIEW

File: fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java [hunk 1]
```diff
@@ -804,6 +837,7 @@ private void verifyPresenceOfUserTypeHeader(final String id) throws IOException
     }
 
     @Test
+@Ignore
```
[the remark this answers is not in the document]
Comment (awoods) [reply fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: Yes... I am expecting them all to be removed soon :)

File: fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java [hunk 2]
```diff
@@ -86,7 +87,7 @@
     private static final org.apache.jena.graph.Node VCARD_GROUP_NODE = createURI(VCARD_GROUP_VALUE);
     private static final org.apache.jena.graph.Node VCARD_MEMBER_NODE = createURI(VCARD_MEMBER_VALUE);
 
-    @Inject
+    // TODO: Should eventually `Inject`
```
[the remark this answers is not in the document]
Comment (awoods) [reply fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java]: Yes. Spring can only inject if there is an implementation of the interface.

File: fcrepo-http-commons/pom.xml [hunk 3]
```diff
@@ -112,14 +119,18 @@
...
     </dependency>
     <dependency>
       <groupId>org.glassfish.jersey.core</groupId>
       <artifactId>jersey-common</artifactId>
-      <version>${jersey.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.hk2</groupId>
+      <artifactId>hk2-api</artifactId>
+      <version>2.6.1</version>
```
Comment (whikloj) [inline fcrepo-http-commons/pom.xml]: Oops I missed one. We can handle this later or I can submit another PR to this one to define this dependency at the root.

Comment (awoods) [reply fcrepo-http-commons/pom.xml]: Another PR to this PR would be good. They should all get squashed before going into `master` anyways.

**Pull request merged**